lib/new-flatfile.ts

```diff
@@ -0,0 +1,74 @@
+import { FlatfileClient } from "@flatfile/api";
+
+const flatfileClient = () => {
+  const token = process.env.FLATFILE_API_KEY;
+
+  if (!token) {
+    throw new Error("No FLATFILE_API_KEY ENV");
+  }
+
+  return new FlatfileClient({
+    token,
+  });
+};
```

Comment:
you can cut this and just `import api from '@flatfile/api'`. it'll give you an authenticated instance.

Reply:
@dboskovic if we need a different environment for each flow (and different api key per flow?) can we still use this?

Reply:
As long as we can keep access to `FlatfileClient` we should be fine, since we can pass the workflow-specific API key for each flow

Reply:
Started slack convo here: https://flatfile.slack.com/archives/CTVRSGLF8/p1685725327491669